tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -269,27 +269,19 @@
         return create_action("set_spc_transaction_mode", {mode, context});
     };
 
-    window.test_driver_internal.create_virtual_sensor = function(create_parameters, context = null) {
-        console.trace("JV666 > tools/wptrunner/wptrunner/testdriver-extra.js create_virtual_sensor()");
-        console.trace(JSON.stringify(create_parameters));
-        return create_action("create_virtual_sensor", {create_parameters, context});
+    window.test_driver_internal.create_virtual_sensor = function(sensor_type, sensor_params, context = null) {
```

Comment:
Style nit: `context=null` rather than `context = null`. `set_spc_transaction_mode` is the outlier in this file.

Reply:
Fixed in https://github.com/JuhaVainio/WPT_sensors/pull/1/commits/83cc4f75ef04a2699fd59f5ddfe0087a5a509369